### DISCUSSION TITLE
Not able to capture the Newrelic Agent dependency

### DISCUSSION THREAD

**Discussion by rajat2911** [renovatebot/renovate#28200] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

Facing an issue where I have a dependency like below in my docker file . Unable to fetch the dependencies
```
    curl -o /usr/local/tomcat/newrelic/newrelic.jar https://download.newrelic.com/newrelic/java-agent/newrelic-agent/8.9.0/newrelic-agent-8.9.0.jar


```

Using my renovate.json file as 

```{
  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
  extends: [
    'config:best-practices',
  ],
  customManagers: [
    {
      customType: 'regex',
      fileMatch: [
        'Dockerfile$',
      ],
      matchStrings: [
        'curl.*https:\/\/download\.newrelic\.com\/newrelic\/java-agent\/newrelic-agent\/(?<currentValue>\d+\.\d+\.\d+)\/newrelic-agent-\g<currentValue>\.jar',
      ],
      depNameTemplate: 'com.newrelic.agent.java:newrelic-agent',
      datasourceTemplate: 'maven',
    },
  ],
}```

**Comment by secustor** [comment, marked as answer]
RegEx does not allow to have two capture groups with the same name.

Extract the `currentValue` in a separate variable. 
https://docs.renovatebot.com/modules/manager/regex/#regular-expression-capture-groups
